packages/p2p/src/pages/advertiser-page/advertiser-page-profile/advertiser-page-profile-rating.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { observer } from '@deriv/stores';
+import { Localize } from 'Components/i18next';
+import RecommendedBy from 'Components/recommended-by';
+import StarRating from 'Components/star-rating';
+import { useStores } from 'Stores';
+import { getLabelSize } from 'Utils/advertiser';
+import AdvertiserPageProfileRatingLabel from './advertiser-page-profile-rating-label';
+
+const AdvertiserPageProfileRating = () => {
+    const { advertiser_page_store, general_store } = useStores();
+
+    const { advertiser_details_id, counterparty_advertiser_info } = advertiser_page_store;
+    const { advertiser_id, advertiser_info } = general_store;
+    const is_my_advert = advertiser_details_id === advertiser_id;
```

Comment:
i see we check this in a few files, would it be possible to create a computed value using this?